docs/getting-started.md

```diff
@@ -161,6 +161,11 @@ please refer to this [guide](kind.md).
 To deploy Antrea in a [Minikube](https://github.com/kubernetes/minikube) cluster,
 please refer to this [guide](minikube.md).
 
+### Deploying Antrea in Rancher Cluster
+
+To deploy Antrea in a [Rancher](https://github.com/rancher/rancher) cluster,
```

Comment:
Prefer Rancher Managed Cluster

Reply:
I suppose it should be `Rancher managed cluster`, @jainpulkit22  there are a few comments are not addressed, please check https://github.com/antrea-io/antrea/pull/4733/files